discord/threads.py

```diff
@@ -632,6 +640,7 @@ async def edit(
         auto_archive_duration: :class:`int`
             The new duration in minutes before a thread is automatically archived for inactivity.
             Must be one of ``60``, ``1440``, ``4320``, or ``10080``.
+            :class:`ThreadAutoArchiveDuration` can be used alternatively.
```

Comment:
This makes the type a union

Reply:
Both types are covariant of `int` shouldn't that be enough ?